### DISCUSSION TITLE
图片里面有图像报错

### DISCUSSION THREAD

**Discussion by Zhuyunfe** [OleehyO/TexTeller#87] [feedback]
大佬，两个问题，如下图，①图片里面有个图像会报错②去掉图片段落不是很清晰，解析出来是一段了。还可以优化吗？
![1_Page5](https://github.com/user-attachments/assets/e689fb66-a729-4b84-b9fd-e8d275bdea1a)

**Comment by OleehyO** [comment]
TexTeller不能用来直接识别一整页的内容。使用段落识别可以，但是会失去版面信息（除非再结合其它版面识别的模型）。
